ui/src/app/routing/app-routing.tsx

```diff
@@ -295,7 +295,11 @@ export const AppRoutingComponent: React.FunctionComponent<
                   guards={[signInGuard, disabledGuard(isUserDisabledInDb)]}
                 >
                   {!redirectToTOSPage && (
-                    <SignedInPage intermediaryRoute={true} routeData={{}} />
+                    <SignedInPage
+                      intermediaryRoute={true}
+                      routeData={{}}
+                      getAccessToken={currentAccessToken}
```

Comment:
I don't love having to pass this through and would appreciate alternative suggestions.